bsp/boards/openmote-b/board_info.h

```diff
@@ -59,36 +59,36 @@
 
 //===== IEEE802154E timing
 
-//#define SLOTDURATION_10MS // by default, we use 10ms time slot
+// #define SLOTDURATION_10MS // by default, we use 10ms time slot
 
 #ifdef SLOTDURATION_10MS
-// time-slot related
-#define PORT_TsSlotDuration                 328   // counter counts one extra count, see datasheet
-// execution speed related
-#define PORT_maxTxDataPrepare               10    //  305us (measured  82us)
-#define PORT_maxRxAckPrepare                10    //  305us (measured  83us)
-#define PORT_maxRxDataPrepare                4    //  122us (measured  22us)
-#define PORT_maxTxAckPrepare                10    //  122us (measured  94us)
-// radio speed related
-#ifdef L2_SECURITY_ACTIVE
-#define PORT_delayTx                        14    //  366us (measured xxxus)
-#else
-#define PORT_delayTx                        12    //  366us (measured xxxus)
-#endif
-#define PORT_delayRx                         0    //    0us (can not measure)
-// radio watchdog
-#else
-// time-slot related
-#define PORT_TsSlotDuration                 492   // counter counts one extra count, see datasheet
-// execution speed related
-#define PORT_maxTxDataPrepare               66    // 2014us (measured 746us)
-#define PORT_maxRxAckPrepare                30    //  305us (measured  83us)
-#define PORT_maxRxDataPrepare               33    // 1007us (measured  84us)
-#define PORT_maxTxAckPrepare                32    //  305us (measured 219us)
-// radio speed related
-#define PORT_delayTx                        42    //  214us (measured 219us)
-#define PORT_delayRx                        0     //    0us (can not measure)
-// radio watchdog
+    // time-slot related
+    #define PORT_TsSlotDuration                 328   // counter counts one extra count, see datasheet
+    // execution speed related
+    #define PORT_maxTxDataPrepare               10    //  305us (measured  82us)
+    #define PORT_maxRxAckPrepare                10    //  305us (measured  83us)
+    #define PORT_maxRxDataPrepare                4    //  122us (measured  22us)
+    #define PORT_maxTxAckPrepare                10    //  122us (measured  94us)
+    // radio speed related
+    #ifdef L2_SECURITY_ACTIVE
+    #define PORT_delayTx                        14    //  366us (measured xxxus)
+    #else
+    #define PORT_delayTx                        12    //  366us (measured xxxus)
+    #endif
+    #define PORT_delayRx                         0    //    0us (can not measure)
+    // radio watchdog
+#else // 40 ms
```

Comment:
The timing below depends on which radio is used by openmote-b, those timing should be moved to radio.h files can call dynamically. 

I recommend to use  a fixed timing for FSK modulation, and use the timing variable for 2.4GHz and OQPSK 800kb/s as well. Only delayTx timing need to be adjust to each three PHY layers. With this, the timing part would be much easier.